docs/articles/patching-edgecases.html

```diff
@@ -1,4 +1,4 @@
-﻿<!DOCTYPE html>
```

Comment:
This file will be overwritten by the CI bot that auto generates the documentation from the above markdown file.